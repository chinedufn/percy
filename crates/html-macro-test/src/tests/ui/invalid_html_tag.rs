extern crate virtual_dom_rs;
use virtual_dom_rs::prelude::*;

// Used a tag name that does not exist in the HTML spec
fn main() {
    html! {
        <invalidtagname></invalidtagname>
    };
}
