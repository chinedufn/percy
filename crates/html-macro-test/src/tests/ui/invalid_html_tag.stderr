error: invalidtagname is not a valid HTML tag.

If you are trying to use a valid HTML tag, perhaps there's a typo?

If you are trying to use a custom component, please capitalize the component name.

custom components: https://chinedufn.github.io/percy/html-macro/custom-components/index.html
 --> $DIR/invalid_html_tag.rs:7:10
  |
7 |         <invalidtagname></invalidtagname>
  |          ^^^^^^^^^^^^^^
